packages/embed/src/lib/nfid.ts

```diff
@@ -222,6 +222,33 @@ export class NFID {
     return response.result;
   }
 
+  async requestTransferNFT({
+    receiver,
+    tokenId,
+    sourceAddress,
+  }: {
+    receiver: string;
+    tokenId: string;
+    sourceAddress: string;
+  }) {
+    console.log('NFID.requestTransferNFT');
+    if (!NFID.nfidIframe) throw new Error('NFID iframe not instantiated');
+    showIframe();
+    const iframe = getIframe();
+    const response = await request(iframe, {
```

Comment:
Please type the response here:
https://github.com/internet-identity-labs/sdk-ts/blob/8456babf79b2145014870f3b0f84bc12fbb74c6d/packages/embed/src/lib/postmsg-rpc.ts#L53-L63

Reply:
And, would be nice to stick to the interface standard:
https://github.com/internet-identity-labs/sdk-ts/blob/8456babf79b2145014870f3b0f84bc12fbb74c6d/packages/embed/src/lib/postmsg-rpc.ts#L17-L19

Reply:
Fungible and non-fungible have the same response interface

Reply:
got it. But the return type is still wrong. Should be `result`